tooling/eslint-plugin/TROUBLESHOOTING.md

```
@@ -0,0 +1,29 @@
+
+## Known Issues
+
+<details>
+<summary><h3>Plugin "@typescript-eslint" was conflicted</h3></summary>
+If you're using the ESLint Plugin inside a Create React App project, you might encounter a stacktrace like below.
+
+```bash
+[eslint] Plugin "@typescript-eslint" was conflicted between ".eslintrc.cjs » plugin:@wso2/react » plugin:@wso2/typescript#overrides[0]" and ".eslintrc.cjs » plugin:@wso2/react » plugin:@wso2/typescript » eslint-config-airbnb-typescript/base » /<PATH>/node_modules/.pnpm/eslint-config-airbnb-typescript@17.0.0_mnxwss6wgpp5zz53fmtbuvhzim/node_modules/eslint-config-airbnb-typescript/lib/shared.js".
+```
+
+#### Reason
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)